hs_core/tests/api/rest/test_resmap.py

```diff
@@ -0,0 +1,145 @@
+import os
+import json
+import tempfile
+import shutil
+from rest_framework import status
+from rdflib import Graph, term
+
+from hs_core.hydroshare import resource
+from .base import ResMapTestCase
+
+
+class TestResourceMap(ResMapTestCase):
```

Comment:
This is an excellent idea. Note that I didn't do a complete job; I just tested that some salient features are there. Testing resourcemetadata.xml is a much bigger job. However, the main heavy lifting is simply calling Graph.parse, which certifies the metadata as RDF compliant unless it throws an exception.